lib/uplinkc/testdata/object_test.c

```diff
@@ -0,0 +1,106 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+#include <string.h>
+#include <stdlib.h>
+
+#include "require.h"
+#include "uplink.h"
+#include "helpers.h"
+
+void handle_project(ProjectRef project);
+
+int main(int argc, char *argv[]) {
+    ProjectOptions opts = {{0}};
+    memcpy(&opts.key, "hello", 5);
+
+    with_test_project(&handle_project, &opts);
+}
+
+void handle_project(ProjectRef project) {
+    char *_err = "";
+    char **err = &_err;
+
+    char *bucket_name = "TestBucket";
```

Comment:
we should start using names that fits future restrictions e.g. `test-bucket`

Reply:
:+1: